packaging/osx/buildpkg.sh

```diff
@@ -4,9 +4,17 @@ set -e
 
```

Comment:
Since we're editing this file, can we change this to `set -Eeuo pipefail` it is a more strict "fail fast" option set.

Reply:
Done, thanks